### PR TITLE
Document get possible statuses companies

### PR DESCRIPTION
**Pull request by timaccelo**
Documented get end point that provides possible company statuses.

### CODE REVIEW

File: source/includes/endpoints/_companies.md [hunk 1]
```diff
@@ -424,6 +424,42 @@ The response will be a single status object with its default fields and any addi
 
 
 
+### Get Company Statuses
```
Comment (matthewhughes112) [inline source/includes/endpoints/_companies.md]: '_List_ Company Statuses'  would be a more standard name here

File: source/includes/endpoints/_companies.md [hunk 2]
```diff
@@ -424,6 +424,42 @@ The response will be a single status object with its default fields and any addi
...
+
+```shell
+curl -X get \
+ https://{deployment}.api.accelo.com/api/v0/companies/statuses \
+  -H 'authorization: Bearer {access_token}'
+```
+
+`GET /companies/statuses`
+
+This request returns the possible [statuses](#statuses) that[companies](#the-company-object) can have.
```
Comment (matthewhughes112) [inline source/includes/endpoints/_companies.md]: Description here could be simplified to e.g.
> Returns a list of company statuses

Comment (matthewhughes112) [inline source/includes/endpoints/_companies.md]: missing space between 'that' and '['

File: source/includes/endpoints/_companies.md [hunk 3]
```diff
@@ -424,6 +424,42 @@ The response will be a single status object with its default fields and any addi
...
+
+#### Configuring the Response
+
+This request supports requesting additional fields and linked objects from the [status object](#statuses) using the
+[`_fields`](#configuring-the-response-fields) parameter.
+
+
+#### Handling The Response
+
+The response will be an array of status objects with its default fields and any additional fields requested via `_fields`.
```
Comment (matthewhughes112) [inline source/includes/endpoints/_companies.md]: typo: 'its' should be 'their'

Comment (timaccelo) [reply source/includes/endpoints/_companies.md]: Updated 4e6d7be

File: source/includes/endpoints/_companies.md (hunk 2 above)
Comment (timaccelo) [reply source/includes/endpoints/_companies.md]: Updated in commit 4e6d7be

Comment (timaccelo) [reply source/includes/endpoints/_companies.md]: Updated in Commit 4e6d7be

File: source/includes/endpoints/_companies.md (hunk 1 above)
Comment (timaccelo) [reply source/includes/endpoints/_companies.md]: Updated in commit 4e6d7be

File: source/includes/endpoints/_companies.md [hunk 4]
```diff
@@ -424,7 +424,7 @@ The response will be a single status object with its default fields and any addi
 
 
 
-### Get Company Statuses
+### List Company Statuses
```
Comment (matthewhughes112) [inline source/includes/endpoints/_companies.md]: The standard is a blank newline after each heading (....I should really document these standards)

**Pull request merged**